### ISSUE TITLE
Continuity only connect vanilla blocks.

### ISSUE THREAD

**Issue by Primia** [Sinytra/Connector#415]
### Describe the bug

Continuity only connect vanilla blocks.

### Steps to reproduce

Continuity only connect vanilla blocks.,it cant support mod resources pack(CTM  resources pack).

### Additional context

Is there any way let Continuity to use resource pack? Identify the CTM resource pack for mod blocks. not vanilla blocks.

**Comment by Su5eD** [comment]
Can you please link the resource pack you're using?

**Comment by Primia** [comment]
resource pack(Contains the mod block resource and the vanilla block resource):
https://www.curseforge.com/minecraft/texture-packs/yuushya-16x
mod：
https://www.curseforge.com/minecraft/mc-mods/yuushya-townscape
Dependencies：
https://www.curseforge.com/minecraft/mc-mods/architectury-api

**Comment by Primia** [comment]
same issues
https://github.com/Sinytra/Connector/issues/422

**Comment by Su5eD** [comment]
After investigating the issue, I found out Continuity builds a map of model IDs to
all registered blocks in its client initializer (`ModelWrappingHandler#createBlockStateModelIdMap`).
However, this is too soon as it happens before blocks of Forge mods are registered. As a results, CTM models will not
be created for Forge mod blocks.

To fix this issue, I added a patch for Continuity to Connector Extras, which moves the block state map initialization from Fabric's client entrypoint to FML's client setup event, which is fired after registration has completed.
